notes/announce-0.6.md

```diff
@@ -0,0 +1,63 @@
+Subject: [ANNOUNCE] jsCoq and SerAPI releases
+
+Dear Coq users and developers,
+
+we are happy to announce the releases of jsCoq 0.9.3 and SerAPI 0.6.1
+for Coq 8.9.
+
+JsCoq and SerAPI are free software, please don't hesitate to contribute at:
+
+- https://github.com/ejgallego/jscoq
+- https://github.com/ejgallego/coq-serapi
+
+## JsCoq 0.9:
+
+JsCoq allows to interact with Coq developments as self-contained HTML,
```

Comment:
```suggestion
JsCoq allows interacting with Coq developments as standard-compliant HTML documents using a browser.
```

Reply:
Maybe it should be "Coq documents", I think we are trying to push that terminology?